### PR TITLE
AST-3288 - Fix: Customiser not loading when all-in-one-schemaorg-rich-snippets plugin is active.

### PR DESCRIPTION
**Pull request by yssambare12**
- Customiser not loading when the all-in-one-schemaorg-rich-snippets plugin is active.

### CODE REVIEW

File: init.php [hunk 1]
```diff
@@ -588,7 +588,7 @@ function bsf_editor_footer_scripts() { ?>
 function bsf_force_send( $args ) {
 
 	if ( ! isset( $_GET['bsf_file_upload_nonce'] ) || ! wp_verify_nonce( $_GET['bsf_file_upload_nonce'], 'ajax_nonce' ) ) {
-		return;
+		return $args;
```
Comment (imnavanath) [inline init.php:591]: Please add a changelog for it,.

Comment (yssambare12) [reply init.php:591]: Added.

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 **Tags:** schema markup, structured data, rich snippets, schema.org, Microdata, schema
 **Requires at least:** 3.7
 **Tested up to:** 6.2
```
Comment (imnavanath) [inline README.md:6]:
```suggestion
**Tested up to:** 6.2.2
```

File: readme.txt [hunk 3]
```diff
@@ -4,7 +4,7 @@ Donate link: https://www.paypal.me/BrainstormForce
 Tags: schema markup, structured data, rich snippets, schema.org, Microdata, schema
 Requires at least: 3.7
 Tested up to: 6.2
```
Comment (imnavanath) [inline readme.txt:6]:
```suggestion
Tested up to: 6.2.2
```

File: README.md (hunk 2 above)
Comment (yssambare12) [reply README.md]: Updated

**Review by imnavanath — APPROVED** [review]

**Pull request merged**